api/v1beta1/hyperconverged_types.go

```diff
@@ -757,6 +760,12 @@ type ApplicationAwareConfigurations struct {
 	AllowApplicationAwareClusterResourceQuota bool `json:"allowApplicationAwareClusterResourceQuota,omitempty"`
 }
 
+type HigherWorkloadDensityConfiguration struct {
+	// MemoryOvercommitRatio defines the ratio of memory overcommit for the vitrual machine workloads
+	// The ratio can be used in order to increase virtual machine density
+	MemoryOvercommitRatio *string `json:"memoryOvercommitRatio,omitempty"`
```

Comment:
Please use CRD annotation for the default value; e.g.
```suggestion
	// +kubebuilder:default=1.5
	MemoryOvercommitRatio *string `json:"memoryOvercommitRatio,omitempty"`
```